compiler/ballerina-backend-jvm/src/main/ballerina/compiler_backend_jvm/jvm_cast_gen.bal

```diff
@@ -37,6 +37,9 @@ function generateCheckCast(jvm:MethodVisitor mv, bir:BType sourceType, bir:BType
     } else if (targetType is bir:BTypeNil) {
         checkCast(mv, targetType);
         return;
+    } else if (targetType is bir:BTableType) {
```

Comment:
Shall we add this condition to the previous else if block?

Reply:
First few types are like the basic types there. I feel like moving the this condition to the end is more suitable.. WDYT? @vinok88